api/workAllocation/index.ts

```diff
@@ -109,14 +103,17 @@ export async function getAllCaseWorkers(req: EnhancedRequest, res: Response, nex
 }
 
 export async function retrieveAllCaseWorkers(req: EnhancedRequest, res: Response): Promise<Caseworker[]> {
+  if (req.session && req.session.caseworkers) {
+    return req.session.caseworkers;
```

Comment:
Shouldn't this return a `Promise` with a `Caseworker[]` payload? 🤔

Reply:
Oh, no, ignore that - I hadn't appreciated that the `async` will automatically wrap it in a `Promise` if it doesn't return one. Cool cool cool. 😄